docs/visual-testing/_partials/_fullpage-js.md

```diff
@@ -1,18 +1,33 @@
-import FullPageDescription from './_fullpage-description.md';
 import FullPageLimit from './_fullpage-limit.md';
 
-<FullPageDescription />
+By default, only the current viewport is captured when `.sauceVisualCheck` is used. You can opt in to capturing the entire page by using the `fullPage` option. It will capture everything by scrolling and stitching multiple screenshots together.
```

Comment:
idea:

worth adding - we recommend using full page screenshots only for tests that require full page screenshot. Otherwise your test execution might be longer.